xml/common_intro_lifecycle_support_i.xml

```diff
@@ -0,0 +1,214 @@
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term>L1</term>
+    <listitem>
+     <para>
+      Problem determination, which means technical support designed to provide
+      compatibility information, usage support, ongoing maintenance,
+      information gathering and basic troubleshooting using available
+      documentation.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>L2</term>
+    <listitem>
+     <para>
+      Problem isolation, which means technical support designed to analyze
+      data, reproduce customer problems, isolate problem area and provide a
+      resolution for problems not resolved by Level&nbsp;1 or prepare for
+      Level&nbsp;3.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>L3</term>
+    <listitem>
+     <para>
+      Problem resolution, which means technical support designed to resolve
+      problems by engaging engineering to resolve product defects which have
+      been identified by Level&nbsp;2 Support.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   For contracted customers and partners, &productname; is delivered with L3
+   support for all packages, except for the following:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Technology Previews
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Sound, graphics, fonts and artwork.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Packages that require an additional customer contract.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Some packages shipped as part of the module <emphasis>Workstation
+     Extension</emphasis> are L2-supported only.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Packages with names ending in <package>-devel</package> (containing header
+     files and similar developer resources) will only be supported together
+     with their main packages.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   &suse; will only support the usage of original packages. That is, packages
+   that are unchanged and not recompiled.
+  </para>
+ </sect2>
+
+ <sect2>
+  <title>Technology Previews</title>
+   <para>
+    Technology previews are packages, stacks, or features delivered by &suse;
+    to provide insights into upcoming innovations. These previews are still in
+    development, but are included for your convenience to give you the chance to
+    test new technologies within an enterprise environment. We would appreciate
+    your feedback! If you test a technology preview, please contact your
+    &suse; representative and let them know about your experience and use cases.
+    Your input is helpful for future development.
+   </para>
+   <para>
+    However, technology previews come with the following limitations:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Technology previews may be functionally incomplete, unstable, or in other ways
```

Comment:
this somewhat repeats what you say above, cf "these previews are still in development". maybe remove one of those mentions of that fact?

Reply:
Merged this part of the sentence into the first listitem, where it fits better.